examples/core/instancing/app.js

```diff
@@ -18,6 +18,10 @@ A luma.gl <code>Cube</code>, rendering 65,536 instances in a
 single GPU draw call using instanced vertex attributes.
 `;
 
+function getDevicePixelRatio() {
```

Comment:
I think the animationLoop provides this as a param

Reply:
It provides the true/false param, but not the device pixel ratio itself: https://github.com/uber/luma.gl/blob/master/modules/core/src/core/animation-loop.js#L286-L308

(Maybe it should?)

Reply:
Yes, I think it should, in fact the ratio can very per frame (e.g. when zooming into a web page) so why not send it in every frame?